CONTRIBUTORS

```diff
@@ -27,15 +27,15 @@ bastienbc <bastien.barbe.creuly@gmail.com>
 Bob Killen <killen.bob@gmail.com>
 Brad Fitzpatrick <bradfitz@golang.org>
 Bruce Downs <bruceadowns@gmail.com>
-Cédric Blomart <cblomart@gmail.com>
```

Comment:
btw, this diff seems to be a difference with `sort` between Linux and Mac.

Reply:
yeah, I noticed as well, will investigate.

Reply:
I standardized `scripts/contributors.sh` on GNU sort, but running it on Mac does not change the outcome... interesting...